### PR TITLE
Fix vertical nav bar issue

### PR DESCRIPTION
**Pull request by saqibameen**
Provides a fix for the #27

### CODE REVIEW

**Comment by saqibameen** [comment]
> Actually `updateLines` is called upon any change to the `posts` object in the store. You don't have to actually call `fetchNextFeedPage` but rather add more posts. Once the `posts` object changes the `updateLines` will be called.

_Originally posted by @idoshamun in https://github.com/dailydotdev/daily-apps/issues/27#issuecomment-607177560_

Following on our conversation on the issue page. Here's what I wrote in the test: 

```js
  const wrapper = mount(DaHome, { store, localVue });

// 1 [] - no post

  feed.state.posts = [...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA, ...POSTS_DUMMY_DATA];

  console.log(wrapper.vm.lineNumbers, wrapper.vm.posts); // 1 [ // all posts]
```

I see that posts are added but the lineNumbers isn't updating. While it is working fine for me in the browser. I am not sure what I'm doing wrong here. 

Ignore console.log, posts population, I am just trying to get it working here. Will update after that.

**Review by idoshamun — CHANGES_REQUESTED** [review]
Thanks for your contribution!
I'm not sure exactly what is the fix as currently, I can see only one change.
Did you forget by any chance to push some commits?

File: packages/components/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "dialog-polyfill": "^0.5.3",
     "dompurify": "^2.1.1",
     "lazysizes": "^5.2.2",
-    "vue-context": "^3.4.0",
+    "vue-context": "^4.1.1",
```
[the remark this answers is not in the document]
Comment (saqibameen) [reply packages/components/package.json]: They were probably auto bumped up. I didn't do it manually. Reverted.

**Comment by saqibameen** [comment]
> Thanks for your contribution!
> I'm not sure exactly what is the fix as currently, I can see only one change.
> Did you forget by any chance to push some commits?

The `Home.vue` was watching for `posts` to call `updateLines()` but `posts` was not imported from store into `Home.vue`. That was the only issue. My guess is probably during the refactoring/updates, `posts` variable was moved to DaFeed and removed from Home.vue. Adding it fixes the issue.

**Comment by idoshamun** [comment]
Oh, this is perfect! Thanks for the explanation.
So should we wait for the test then? You might need to call `$nextTick` after changing the posts variable.

**Comment by saqibameen** [comment]
> Oh, this is perfect! Thanks for the explanation.
> So should we wait for the test then? You might need to call `$nextTick` after changing the posts variable.

Yeah, trying to figure out the test part. Might learn a thing or two about testing and make it future proof. 

I tried calling `$nextTick` once. Didn't change anything.

**Comment by idoshamun** [comment]
Hey @saqibameen,

Can I assist in any way so we can proceed with this PR?

**Comment by idoshamun** [comment]
I'm closing this issue as we redesign the sidebar, so this is no longer relevant.
Thanks for the efforts so far.

**Pull request closed without merging**